README_ja.md

```
@@ -104,6 +105,106 @@ docker compose up -d
+      properties:
+        fileId:
+          description: ""
+          type: string
+      required:
+        - fileId
+      type: object
+```
+
+推奨するワークフロー:
+
+1. サーバーの設定に `tools.file` を追加し（[`mcpServers.<name>.tools`](#mcpservernametools) 参照）、`manifold openapi generate -c config` を実行する
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)